hack/e2e.go

```diff
@@ -327,44 +327,11 @@ func finishRunning(stepName string, cmd *exec.Cmd) bool {
 	return result
 }
 
-func printBashOutputs(headerprefix, lineprefix, stdout, stderr string, escape bool) {
```

Comment:
I like this but it's unrelated to the exit status of `-test`, so can we please keep it in a separate commit?